bot/main.py

```diff
@@ -51,9 +54,7 @@ def get_realname_from_discordid(user_id):
 
 
 async def send_link(ctx):
-    await ctx.send(
-        f"<{PORTAL_LINK}>\nSign in through our portal, and try again."
-    )
+    await ctx.send(f"<{PORTAL_LINK}>\nSign in through our portal, and try again.")
```

Comment:
Unrelated change.

Reply:
These are made by `black`. but I'll revert them ig.

Reply:
You can do that as a separate patch if you like, but it's still unrelated to this patch.